package.json

```diff
@@ -32,7 +33,7 @@
   "dependencies": {
     "es6-collections": "0.x",
     "skatejs-named-slots": "1.x",
-    "webcomponents.js": "webcomponents/webcomponentsjs#b77ca74"
```

Comment:
The former. You mean like skate has or like you have? Either way I tend to
agree. The skates fork has legitimate PRs in against specd behaviour. That
said, I'm going to spend some more time trying to use webreflection's
polyfill instead as it is the most feature complete impl and versioned.

On Sat, 27 Aug 2016, 03:48 Joshua Nelson <notifications@github.com> wrote:

> In package.json
> <https://github.com/skatejs/web-components/pull/6#discussion_r76459291>:
>
> > @@ -32,7 +33,7 @@
> >    "dependencies": {
> >      "es6-collections": "0.x",
> >      "skatejs-named-slots": "1.x",
> > -    "webcomponents.js": "webcomponents/webcomponentsjs#b77ca74"
>
> Oh really? So when you say "our v1 branch", do you mean
> https://github.com/skatejs/webcomponentsjs/tree/v1 ? I also noticed
> https://github.com/skatejs/webcomponentsjs/tree/skate-v1 . Either way,
> I'd be very happy to stick to not fork it like this.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/skatejs/web-components/pull/6/files/5b1b5c79b018aab352ac8128b4e14b3a9fcd197b#r76459291>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAIVbGdu1ChIij59HVJzqhafF8nvkACQks5qjyblgaJpZM4JtUNA>
> .
>